plugin-server/src/utils/db/db.ts

```diff
@@ -465,6 +482,84 @@ export class DB {
         }
     }
 
+    public async getPersonIdThroughCache(teamId: number, distinctId: string): Promise<number | null> {
+        if (!this.personInfoCachingEnabledTeams.has(teamId)) {
+            return null
+        }
+        const personId = await this.redisGet(this.getPersonIdCacheKey(teamId, distinctId), null)
+        if (personId) {
+            this.statsd?.increment(`person_info_cache.hit`, { lookup: 'person_id', team_id: teamId.toString() })
+            return personId as number
```

Comment:
that's a smell - you should try to never use `as`. What it does is trick the TS compiler, it doesn't cast. You should do `Number(personId)`